sklearn/preprocessing/_data.py

```diff
@@ -3525,9 +3526,10 @@ def _neg_log_likelihood(lmbda):
 
         # the computation of lambda is influenced by NaNs so we need to
         # get rid of them
-        x = x[~np.isnan(x)]
+        #x = x[~np.isnan(x)]
         # choosing bracket -2, 2 like for boxcox
-        return optimize.brent(_neg_log_likelihood, brack=(-2, 2))
+        #return optimize.brent(_neg_log_likelihood, brack=(-2, 2))
```

Comment:
Remove this comment and the above comment as they should be in the new function now